### PR TITLE
Add getFutureWeekDate to datePickerService

### PR DESCRIPTION
**Pull request by cjmochrie**

### CODE REVIEW

File: src/form/ws-date-picker/ws-date-picker.service.js [hunk 1]
```diff
@@ -43,6 +43,18 @@ export default class WsDatePickerService {
     return this.getFutureDate(numbDays).toString();
   }
 
+  getFutureWeekDate(numbDays) {
```
Comment (benjyblack) [inline src/form/ws-date-picker/ws-date-picker.service.js]: `numbDays`

https://www.youtube.com/watch?v=kXYiU_JCYtU

Comment (benjyblack) [inline src/form/ws-date-picker/ws-date-picker.service.js]: Feel like this method name could be a bit more clear... It's hard to understand what it's doing.

It's more like `getNextWeekStart(numDaysOffset)`

File: src/form/ws-date-picker/ws-date-picker.service.test.js [hunk 2]
```diff
@@ -51,6 +51,16 @@ describe('wsDatePickerService', () => {
     });
   });
 
+  describe('getFutureWeekDateAsString', () => {
+    it('returns the current date + the provided number of days', () => {
+      expect(moment(wsDatePickerService.getFutureWeekDateAsString(5)).isSame(moment().add(5, 'day'), 'day')).toBe(true);
```
Comment (AlexisDeschamps) [inline src/form/ws-date-picker/ws-date-picker.service.test.js]: Maybe break this into multiple lines, it is already over 100.

**Comment by cjmochrie** [comment]
Closing as this is the wrong repo for this thing.

**Pull request closed without merging**